sw/device/tock/BUILD

```diff
@@ -9,47 +9,47 @@ package(default_visibility = ["//visibility:public"])
 filegroup(
     name = "lowrisc_defs",
     srcs = [
-        "//hw/ip/adc_ctrl/data:adc_ctrl_rust_regs",
-        "//hw/ip/aes/data:aes_rust_regs",
-        "//hw/ip/aon_timer/data:aon_timer_rust_regs",
-        "//hw/ip/csrng/data:csrng_rust_regs",
-        "//hw/ip/edn/data:edn_rust_regs",
-        "//hw/ip/entropy_src/data:entropy_src_rust_regs",
-        "//hw/ip/gpio/data:gpio_rust_regs",
-        "//hw/ip/hmac/data:hmac_rust_regs",
-        "//hw/ip/i2c/data:i2c_rust_regs",
-        "//hw/ip/keymgr/data:keymgr_rust_regs",
-        "//hw/ip/kmac/data:kmac_rust_regs",
-        "//hw/ip/lc_ctrl/data:lc_ctrl_rust_regs",
-        "//hw/ip/otbn/data:otbn_rust_regs",
-        "//hw/ip/otp_ctrl/data:otp_ctrl_rust_regs",
-        "//hw/ip/pattgen/data:pattgen_rust_regs",
-        "//hw/ip/pwm/data:pwm_rust_regs",
-        "//hw/ip/rom_ctrl/data:rom_ctrl_rust_regs",
-        "//hw/ip/rv_core_ibex/data:rv_core_ibex_rust_regs",
-        "//hw/ip/rv_timer/data:rv_timer_rust_regs",
-        "//hw/ip/spi_device/data:spi_device_rust_regs",
-        "//hw/ip/spi_host/data:spi_host_rust_regs",
-        "//hw/ip/sram_ctrl/data:sram_ctrl_rust_regs",
-        "//hw/ip/sysrst_ctrl/data:sysrst_ctrl_rust_regs",
-        "//hw/ip/uart/data:uart_rust_regs",
-        "//hw/ip/usbdev/data:usbdev_rust_regs",
+        "//hw/top:adc_ctrl_rust_regs",
```

Comment:
Could this list of _regs targets be provide by top?

Reply:
Yes we could create a single target that collects all rust_regs into on in `//hw/top/`